GA/config.py

```
@@ -0,0 +1,302 @@
+deviation_range = 0.01  # +/- x% of the reference value for the constraints
+
+# Define the constraint IDs used to construct the constraint functions
+# constraint IDs are structured as a nested list, of the form:
+# [[inequality constraint 1, inequality constraint 2, ...],
+#  [equality constraint 1, equality constraint 2, ...]]
+class InEqConstraintID(IntEnum):
+    """
+    Enumeration of the inequality constraint identifiers for the optimization problem.
+    Each member represents a different inequality constraint that can be used in the genetic algorithm's evaluation.
+    """
+    def _generate_next_value_(name, start, count, last_values):
```

Pylint (3.3.7)

[error] Method '_generate_next_value_' should have "self" as first argument

(E0213)